.github/workflows/deploy.yml

```diff
@@ -7,48 +7,31 @@ on:
 jobs:
```

Comment:
It seems like this job is still set up to run for every commit on master, but I think that would make this CI fail unless we always bump the vcap and vcap_utils versions for every commit since PyPI doesn't let you overwrite versions. Should it instead run only when there's a git tag?

Reply:
poetry has a `version` command. So you can do `poetry version minor` and it bumps the version in the toml. 
Is this something we could integrate into the CI?